src/Network/Network/help/New-AzFirewall.md

```diff
@@ -359,6 +414,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ### System.String
 
+### Microsoft.Azure.Commands.Network.Models.PSVirtualNetwork
+
+### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.Network.Models.PSPublicIpAddress, Microsoft.Azure.PowerShell.Cmdlets.Network, Version=1.8.1.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
It looks like this wasn't updated when the parameter was changed from a List to and array.

Reply:
Done, thank you.